src/controllers/colorjsproxy.h

```diff
@@ -5,23 +5,29 @@
 #include <QScriptEngine>
 #include <QScriptValue>
 
+#include "preferences/colorpalettesettings.h"
 #include "util/color/color.h"
 
-class ColorJSProxy: public QObject {
+class ColorJSProxy final : public QObject {
     Q_OBJECT
   public:
-    ColorJSProxy(QScriptEngine* pScriptEngine);
+    ColorJSProxy(QScriptEngine* pScriptEngine,
```

Comment:
Do we need this class anymore? As far as I can tell it isn't used in this PR and I am not sure what a use case for it would be. I think we only need ColorMapper.

Reply:
Whoops, good catch.